platypus-core/src/test/java/com/platypus/crw/udp/UdpVehicleServerTest.java

```diff
@@ -45,6 +46,8 @@
     
     UdpVehicleService service;
     SimpleBoatSimulator sbs;
+    int port;
```

Comment:
Can we just remove this variable and declare `int port` locally right where we use it?  I don't think the test needs the port number anywhere else.